StudentTrackingSystemApp/configfuncs.py

```diff
@@ -99,11 +99,11 @@ def get_course_type(course_code):
     if temp is not None:
         course_code = temp
 
-    if _is_core(course_code):
-            return 'CORE'
-
     course_type = _validate_tag(course_code)
 
+    if _is_core(course_code) and not _is_exception(course_code, course_type):
+            return "CORE"
+
```

Comment:
Wouldn't a course that's in the core list and an exception still be considered "CORE", as the exceptions are exceptions to the TEs people can take?

Reply:
My understanding is that if its in the exception page, then its not supposed to have a course-type even if its in the core list. 

But this probably won't happen since none of the core courses are in the exception page, its just a "safety" check.

If it needs to be removed I can remove it and push the new changes.

Reply:
Okay, that sounds good. We should reach out to owen about verifying this. I'll get him to take a look before resolving.

Reply:
I'm looking at this in the trello page: 
![image](https://user-images.githubusercontent.com/10579359/142740531-4840eb15-c4a6-4abe-a15e-d54615b17b81.png)

Seems like the exceptions have no bearing on the core courses. I think we should remove the second check. What do you think?

Reply:
Sure I'll remove that check then

Reply:
Elliot is correct that if it is in the exception sheet it should not be considered, however there should never be a core course listed there since the point of the exception list is to simplify the parsing for TE's. So the check is redundant but not incorrect, I would say remove